azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -67,6 +69,7 @@
   private Logger flowLogger = null;
 
   private Appender jobAppender;
+  private Appender kafkaAppender;
```

Comment:
Let's use Optional here since this is a configurable parameter which may or may not be present.